internal/engine/wazevo/backend/isa/amd64/operands.go

```diff
@@ -120,22 +120,22 @@ func newAmodeRipRelative(label backend.Label) amode {
 }
 
 // String implements fmt.Stringer.
-func (a *amode) String() string {
+func (a *amode) format(_64 bool) string {
```

Comment:
why do you need _64 flag for addressing mode? shouldn't it be always 64-bit address on x64?

Reply:
does `mull 123(%edi)` make any sense at all?

Reply:
let's revert this change on amode